src/ripple/core/Config.h

```diff
@@ -275,6 +278,15 @@ class Config : public BasicConfig
     // First, attempt to load the latest ledger directly from disk.
     bool FAST_LOAD = false;
 
+    // When starting rippled with existing database it do not know it has those
+    // ledgers locally until the server naturally tries to backfill. This makes
```

Comment:
This first sentence is a bit odd.  Consider replacing it with something like, "When rippled starts with an existing database, it does not know which ledgers it has locally until the server tries to backfill."

Reply:
Yep, agreed - that was not well written. Substituted your prose.

Reply:
FWIW, if that change was intended for the `[fold] Minor cleanups` commit, it doesn't look like the change make it in.  But no problem if that change is still coming.  Also, not a big deal if it's left as is.

Reply:
Fixed